report/sections/Chapter 10 - Discussion/discussion.tex

```diff
@@ -0,0 +1,40 @@
+This project aims to create a teaching platform to facilitate learning with active feedback among lecturers and students.
+By providing a user-friendly interface for creating and solving exercises, as well as a system for managing and organizing the exercises, the platform seeks to enhance the educational experience for students in the \textit{Programming Paradigms} course.
+In this discussion, we will explore the features and functionality of the platform, as well as its potential impact on the teaching and learning process.
+One of the key features of the platform is its ability to support the creation of exercises, allowing students to submit solution attempts and receive feedback quickly.
+
+The platform's user-friendly interface makes it easy for lecturers to create and organize exercises, and to specify the tests that evaluate student answers. This not only allows lecturers to tailor exercises to the specific needs and abilities of their students, but also to provide detailed feedback on student performance, which can help students to improve their understanding and skills.
+We conducted a usability test with students from the \textit{Programming Paradigms} course to evaluate the user interface.
+To conduct this usability test, we provided them with a questionnaire based on the \textit{System Usability Scale} (SUS).
+The participants of the usability test gave the platform a high SUS score, indicating that the platform is usable to such a degree that it could be deployed for use in the class room. The participants responses particularly indicated that the platform's ability to display useful error messages on code submissions was very helpful.
```

Comment:
```suggestion
The participants of the usability test gave the platform a high SUS score, indicating that the platform is usable to such a degree that it could be deployed for use in the classroom. The participants responses particularly indicated that the platform's ability to display useful error messages on code submissions was very helpful.
```